dipy/viz/utils.py

```diff
@@ -228,3 +228,251 @@ def lines_to_vtk_polydata(lines, colors=None):
+
+    Parameters
+    ----------
+    polydata : vtkPolyData
+    
+    
+    Returns
+    -------
+    output : normals, represented as 2D ndarrays (Nx3)
+             None (if no normals in the vtk polydata)
+    """
+    vtk_normals = polydata.GetPointData().GetNormals()
+    if vtk_normals is None:
+        return None
+    else:
+        return ns.vtk_to_numpy(vtk_normals)
+
+
+def get_polydata_colors(polydata):
+    """ get points color (ndarrays Nx3 int) from a vtk polydata
+
+    Parameters
+    ----------
+    polydata : vtkPolyData
+    
+    
+    Returns
+    -------
+    output : colors, represented as 2D ndarrays (Nx3)
+             None (if no normals in the vtk polydata)
+    """
+    vtk_colors = polydata.GetPointData().GetScalars()
+    if vtk_colors is None:
+        return None
+    else:
+        return ns.vtk_to_numpy(vtk_colors)
+
+
+##########################################
+# Set PolyData properties with Numpy array
+##########################################
+def set_polydata_triangles(polydata, triangles):
+    """ set polydata triangles with a numpy array (ndarrays Nx3 int) 
+
+    Parameters
+    ----------
+    polydata : vtkPolyData
+    triangles : triangles, represented as 2D ndarrays (Nx3)
+    """
+    vtk_triangles = np.hstack(np.c_[np.ones(len(triangles)).astype(np.int) * 3, triangles])
+    vtk_triangles = ns.numpy_to_vtkIdTypeArray(vtk_triangles, deep=True)
+    vtk_cells = vtk.vtkCellArray()
+    vtk_cells.SetCells(len(triangles), vtk_triangles)
+    polydata.SetPolys(vtk_cells)
+    return polydata
+
+def set_polydata_vertices(polydata, vertices):
+    """ set polydata vertices with a numpy array (ndarrays Nx3 int) 
+
+    Parameters
+    ----------
+    polydata : vtkPolyData
+    vertices : vertices, represented as 2D ndarrays (Nx3)
+    """
+    vtk_points = vtk.vtkPoints()
+    vtk_points.SetData(ns.numpy_to_vtk(vertices, deep=True))
+    polydata.SetPoints(vtk_points)
+    return polydata
+
+def set_polydata_normals(polydata, normals):
+    """ set polydata normals with a numpy array (ndarrays Nx3 int) 
+
+    Parameters
+    ----------
+    polydata : vtkPolyData
+    normals : normals, represented as 2D ndarrays (Nx3) (one per vertex)
+    """
+    vtk_normals = ns.numpy_to_vtk(normals, deep=True)
+    polydata.GetPointData().SetNormals(vtk_normals)
+    return polydata
+
+def set_polydata_colors(polydata, colors):
+    """ set polydata colors with a numpy array (ndarrays Nx3 int) 
+
+    Parameters
+    ----------
+    polydata : vtkPolyData
+    colors : colors, represented as 2D ndarrays (Nx3) 
+                colors are uint8 [0,255] RGB for each points
+    """
+    vtk_colors = ns.numpy_to_vtk(colors, deep=True, array_type=vtk.VTK_UNSIGNED_CHAR)
+    vtk_colors.SetNumberOfComponents(3)
+    vtk_colors.SetName("RGB")
+    polydata.GetPointData().SetScalars(vtk_colors)
+    return polydata
+
+
+##########################################
+# Update/Refresh PolyData
```

Comment:
Same, remove all comments describing functions on top of the functions. If you keep the information in the docstring it suffices.